### DISCUSSION TITLE
Carbone SDK Artifacts (delivrables)

### DISCUSSION THREAD

**Discussion by yelghali** [Green-Software-Foundation/carbon-aware-sdk#46] [General]
Hello team,

proposal  for a conversation on the (Carbone Aware SDK) Artifacts to be published, and how the SDK is going to be consumed :

SDK CLI:
* Built: as Docker image
* Publised as: 
    -  **Docker image in Github Registry (and Docker Hub) ?**
   -  Github Action: 
      - https://github.com/Green-Software-Foundation/carbon-aware-sdk/blob/container-build/.github/workflows/using-carboneaware-githubaction.yml 
      -   https://github.com/Green-Software-Foundation/carbon-aware-sdk/runs/5433377030?check_suite_focus=true

SDK API:
* built as Docker image:
* Published as:
    -  **Docker image in Github Registry (and Docker Hub) ?**
  - API Endpoint (deployed somewhere, requires cloud account / subscription)

SDK lib package (.NET as first language):
 -  built as .Net code (dotnet build / package) 
 -  **Published as nuget package in Github Package (and nuget.org) ?**

SDK Doc: Built and Published as Github Page: https://green-software-foundation.github.io/carbon-aware-sdk/

**Comment by YaSuenag** [comment]
I give +1 to use GitHub Packages to publish the SDK binary because the SDK supports several languages (e.g. Java, JS, .Net, etc...), so GitHub Packages is more suitable.

**Comment by vaughanknight** [comment]
At this point in time we are not committing to binary builds (including packages such as nuget, pip etc) until we have formal confirmation on how to manage, support, and related liability for this
